js/background.js

```diff
@@ -1,201 +1,203 @@
 /* eslint-disable no-unused-vars */
 /* global GithubClient */
-chrome.browserAction.onClicked.addListener((tab) => {
+chrome.action.onClicked.addListener((tab) => {
```

Comment:
This change causes Firefox, that uses MV2, to fail launching the listen1 main page when clicking on the extension icon.

Either reverting this line or changing this line to `(chrome.action||chrome.browserAction).onClicked.addListener((tab) => {` fixes the above problem.